plugin/mutate/source/dextool/plugin/mutate/backend/database/standalone.d

```diff
@@ -292,14 +292,23 @@ struct Database {
 
     /// Add a mutation score to the history table.
     void putMutationScore(const MutationScore score) @trusted {
-        db.run(insert!MutationScoreHistoryTable, MutationScoreHistoryTable(0,
-                score.timeStamp, score.score.get));
+        auto stmt = db.prepare("INSERT OR REPLACE INTO " ~ mutationScoreHistoryTable ~ "
+                (score, time) VALUES (:score, :time);");
+        stmt.get.bind(":score", score.score.get);
+        stmt.get.bind(":time", score.timeStamp.toISOExtString().replace("T",
```

Comment:
dont forget to call toUTC before stringification because otherwise the timeStamp is in local clock

Reply:
I call it in test_mutant/package.d on line 1121, so it works with UTC time from the start

Reply:
instead of calling replace you can call toSqliteDateTime in miniorm.